src/telliot_feeds/feeds/eth_usd_30day_volatility.py

```diff
@@ -11,6 +11,7 @@
 from telliot_feeds.sources.price.historical.coingecko_daily import CoingeckoDailyHistoricalPriceSource
 from telliot_feeds.sources.price.historical.cryptowatch import CryptowatchHistoricalPriceService
 from telliot_feeds.sources.price.historical.cryptowatch import CryptowatchHistoricalPriceSource
+from telliot_feeds.sources.price.historical.kraken_ohlc import KrakenHistoricalPriceSourceOHLC
 from telliot_feeds.sources.price_aggregator import PriceAggregator
 from telliot_feeds.utils.log import get_logger
 
```

Comment:
- Move `get_volatility()` to the relevant data source file instead.
- Not seeing a new query file for `DailyVolatility`. Am I missing that somewhere?
- Will hard-coding in `midnight` restrict the datafeed to only report for for one 30-day period over and over again? Is that the intended behavior or is it supposed to update the 30-day window every day?

Reply:
https://github.com/tellor-io/telliot-feeds/blob/add-30day-eth/usd-volatility-support/src/telliot_feeds/queries/daily_volatility.py